docker/content-build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cimg/python:3.11-node
+FROM cimg/python:3.10-node
```

Comment:
why did this get updated at all? Isnt this devonly?

Reply:
maybe my PR screwed this in some way, because it contained my changes

Reply:
yes its my because of my PR, i forgot to add `continue` statement for dev, so my fault